src/ocean/meta/types/Qualifiers.d

```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+
+    Templates to define types with modified qualifiers based on some input
+    types.
+
```

Comment:
I added mention that some of templates are transition related but the module as a whole is not.